docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:16-alpine3.20
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - my_db:$PGDATA
```

Comment:
Ensure that `$PGDATA` is correctly set in the `.env` file or consider using a default path like `/var/lib/postgresql/data` for the database volume.